### ISSUE TITLE
Error $.okvideo is not a function

### ISSUE THREAD

**Issue by TASimpson** [okfocus/okvideo#42]
I am trying to set this thing up and it seems easy enough...however, I can't seem to get passed this issue. I'm sure I am missing something small, I'm pretty new to all this stuff. But this is set up at the bottom of my HTML page. 

![image](https://cloud.githubusercontent.com/assets/11613404/8222491/dce14ae2-1531-11e5-8944-8bc5e13ffd25.png)

Here is my error: 
![image](https://cloud.githubusercontent.com/assets/11613404/8222510/1826fb60-1532-11e5-9331-f66b84e1cbcf.png)

Thanks again for your help!

**Comment by julescarbon** [comment]
Check your paths, ".." means look one directory up, you probably don't need the ../../../

**Comment by TASimpson** [comment]
I tried that. But no matter if I take them off, leave them on, etc. I still get the $.okvideo is not a function

**Comment by julescarbon** [comment]
Filename might be something other than okvideo.js depending on what you saved.

Don't mean to be pedantic but look at what directory you're serving HTML out of and see if "assets" is a subdirectory of that, and if "js" is a subdirectory of that, and if okvideo.js is inside that, or if these files are somewhere else entirely.  ".." means move one directory up.  If you can "cd" from one directory to another in the terminal you can figure this out, paths and filenames are like terminal 101.

**Comment by TASimpson** [comment]
My assets are not a subdirectory of the HTML page having issues. I have multiple html pages that have their folders. This is because of how we have set up our architecture of our project. So I have one assets folder that they all pull from. I do not have issues with other HTML pages pointing to this one assets folder. 

But I went ahead and created a js subfolder under where my HTML page sits and i was able to get a video to show on the HTML page when I debugged it. However, when I redirected to the home page of my project (set up with angular.js) I got an error again. This looks to be an issue again with the $.okvideo is not a function...this looks like an issue with angular and this plugin working together?? 

![image](https://cloud.githubusercontent.com/assets/11613404/8232193/3683c94c-1593-11e5-886b-5851a8ee166d.png)
